aws/README.md

```diff
@@ -20,10 +20,20 @@ create create new VPC with public internet access and set also the following env
 export VPC_ID="REPLACE_ME"
 export SUBNET_ID="REPLACE_ME"
 ```
-
+   
 - Create a custom AMI based on Ubuntu 20.04 having kata-agent and other dependencies
-        - **NOTE:** If authentication against container image registries is required copy your JSON format credentials
-        file to [podvm/files/](../podvm/files/)auth.json and add also `USE_SKOPEO=true` before `make image`
+- **Setting up authenticated registry support:** 
+  - If authentication against container image registries is required copy your JSON format credentials file to [podvm/files/etc/](../podvm/files/etc/)auth.json 
```

Comment:
Hi, `files/etc/`? it seems copy-files.sh doesn't copy the file from this location, does it? maybe i missed something..

Reply:
Hi Snir, as discussed earlier the auth.json file isn't needed to be copied into the peer pod vm image, so it doesn't need to go into this exact place, it was just a suggestion for simplicity, but @Jordan9500 maybe we should clarify that, or move the instructions so it is clear that `aa-offline_fs_kbc-resources.json` goes into `podvm/files/etc/`.

Reply:
Hi Snir, 
I have updated the lines to better reflect that the auth.json file is no longer required to be inside the `podvm/files/` directory. And made it clear the offline kbc file should be in the `podvm/files/etc/` location.

Reply:
The aws instructions are currently explains how to set the auth.json file so that it will be used by skopeo in the podvm, AFAIU as long as it is under `podvm/files/etc/ ` it will not be copied by copy-files.sh to the path expected by skopeo.
I think you can just remove this note and mention just to use the user auth.json file, instructions are expected to be changed soon anyway

Reply:
> I think you can just remove this note and mention just to use the user auth.json file, instructions are expected to be changed soon anyway

Hey Snir, sorry, I'm not sure I understand this bit. This PR changes the instructions from using skopeo, to image-rs (with the attestation-agent), so the location of `auth.json` isn't relevant as we won't be using it directly in the peer podvm image any more?

Reply:
right, sorry, i misread, i thought it implies the location must be on this path, LGTM